### PR TITLE
Move dev to gvl

### PR DESCRIPTION
**Pull request by jlqfab**
Updates to playbooks and configs to move dev from usegalaxy.org.au domain to gvl.org.au domain.

### CODE REVIEW

File: host_vars/dev.gvl.org.au.yml [hunk 1]
```diff
@@ -357,14 +357,14 @@ singularity_default_container_id: "/cvmfs/singularity.galaxyproject.org/all/pyth
...
-galaxy_subsite_base_domain: "dev.usegalaxy.org.au"
+galaxy_subsite_base_domain: "dev.gvl.org.au"
 galaxy_subsite_default_welcome: "https://site.usegalaxy.org.au"
 galaxy_subsite_dir: /mnt/galaxy/subsites
 #galaxy_subsite_base_css: '#masthead { background-color: #003399;}'
 galaxy_subsites:
   - name: genome
     brand: Genomes lab
-    iframe: "https://dev-site.usegalaxy.org.au/landing/genome"
+    iframe: "https://dev-site.gvl.org.au/landing/genome"
```
Comment (cat-bro) [inline host_vars/dev.gvl.org.au.yml:367]: this line probably does not need to change

Comment (jlqfab) [reply host_vars/dev.gvl.org.au.yml:367]: dev-site.usegalaxy.org.au is not the same as site.usegalaxy.org.au, which is referenced in line 361. Once dev*.usegalaxy.org.au DNS entries are removed and only dev*.gvl.org.au DNS entries remain, then dev-site.usegalaxy.org.au won't exist.

**Review by cat-bro — APPROVED** [review]

**Comment by cat-bro** [comment]
this hostname variable will need to be overridden somewhere, maybe in group_vars/dev.yml https://github.com/usegalaxy-au/infrastructure/blob/master/group_vars/all.yml#L2

**Comment by neoformit** [comment]
> this hostname variable will need to be overridden somewhere, maybe in group_vars/dev.yml https://github.com/usegalaxy-au/infrastructure/blob/master/group_vars/all.yml#L2

Good catch

**Review by neoformit — APPROVED** [review]
Can't see anything off. I assume you did a regex search for dev*.usegalaxy?

**Comment by jlqfab** [comment]
Thanks @cat-bro I was wondering where the FQDN came from.
Yes @neoformit I used `$ ack dev.*usegalaxy.org.au`

**Review by cat-bro — APPROVED** [review]

**Pull request merged**